starter_code/css/mobile.css

```diff
@@ -0,0 +1,85 @@
+
+#sidebar {
+  max-width: 100%;
+  display: inline-block;
+  vertical-align: top;
+}
+#hero h2 {
+  font-size: 48px;
+  text-align: center;
+  color: #fff;
+  margin: 0 auto;
+  font-weight: 600;
+}
+#benefits ul {
+  list-style: disc;
+  margin: 0 auto;
+}
+
+.container{
+	max-width: 100%;
+	margin: 0 auto;
+}
+#benefits p, #benefits ul {
+  max-width: 100%;
+}
+#posts {
+  max-width: 100%;
+  display: inline-block;
+  margin-right: 0 auto;
+  vertical-align: top;
+}
+#sidebar {
+  max-width: 100%;
+  display: inline-block;
+  vertical-align: top;
+}
+img{
+  max-width: 100%;
+    height: auto;
+}
+p{
+	max-width: 100%;
+}
+blockquote {
+  font-size: 36px;
+  font-weight: 300;
+  font-style: italic;
+  text-align: center;
+  max-width: 100%;
+  margin: 0 auto;
+}
```

Comment:
All these styles that are not contained with a media query (everything above this line within this file) should not go in this file as they are now being applied universally. I recommend moving them to your base stylesheet.